updated the start monit and stop monit tasks in deployment script

G33tha opened the pull request:
Added the monit stop, monit start and remove old artifacts tasks in learning, search, neo4j and logstash deployment scripts.

keshavprasadms commented on ansible/roles/search-service-deploy/tasks/main.yml:
```diff
@@ -40,6 +43,11 @@
+- name: delete existing search artifact
+  become: yes
+  become_user: "{{learner_user}}"
+  file: name={{ search_root }}/search-manager-1.0-SNAPSHOT-dist.zip state=absent
```
@G33tha Please use a varible in defaults with the value being the file name as this is repeating multiple times in role, it would be good to update in one place in case of a change

G33tha replied:
@keshavprasadms above changes have been done. Please review the changes.

G33tha commented:
@maheshkumargangula We have updated the learning, search logstash and neo4j deployment script with additional tasks to Stop monit and start monit service. This addition is to make sure monit should not play any role when deployment happens.

Also, we have added the tasks to remove the old artifatcs file in the server before copying new artifacts into the server to make sure new code deployed . Please review the changes and suggest if any.

maheshkumargangula commented on ansible/roles/learning-service/tasks/main.yml:
```diff
@@ -10,13 +18,18 @@
   become_user: "{{learner_user}}"
   template: src=application.conf.j2 dest=/data/properties/application.conf mode=755 owner={{ learner_user }} group={{ learner_group }}
 
+- name: Delete existing artifact file from tomcat webapps folder
```
We should delete the folder also.

G33tha replied:
task for deleting learning-service folder was alredy there, we are deleting folder before we start tomcat.

G33tha commented:
@PradyumnaNagendra @maheshkumargangula  Please review and approve the PR
